.github/workflows/e2e-tests.yml

```diff
@@ -6,16 +6,19 @@ on:
   pull_request:
 jobs:
   e2e-tests:
+    strategy:
+      matrix:
+        parallel-id: [0, 1, 2, 3]
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v2
       - uses: actions/setup-go@v2
         with:
           go-version: '~1.16'
-      - run: make e2e-local E2E_NODES=2 ARTIFACTS_DIR=./artifacts/
+      - run: make e2e-local E2E_TEST_CHUNK=${{ matrix.parallel-id }} E2E_TEST_NUM_CHUNKS=${{ strategy.job-total }} E2E_NODES=2 ARTIFACTS_DIR=./artifacts/
```

Comment:
Any idea on where I can find some documentation around the `strategy.job-total` syntax?

Reply:
I couldn't find much other than this [mention](https://docs.github.com/en/actions/learn-github-actions/contexts#about-contexts) and this [post](https://github.community/t/get-length-of-strategy-matrix-or-get-all-matrix-options/18342/3). I suspect you'd have to read the actions code to get more info.

Reply:
Nope that's fine - I think it's a bit obvious once you wrap you head around the new e2e system, and was mainly curious.